mlflow/R/mlflow/R/tracking-observer.R

```diff
@@ -0,0 +1,13 @@
+# If one or more external observer(s) are present, then inform them of the
+# event. Otherwise do nothing.
+mlflow_register_tracking_event <- function(event_name, data) {
+  observers <- getOption("MLflowObservers")
+  if (length(observers) > 0) {
+    lapply(
```

Comment:
Should we catch exceptions thrown by observers/log warnings? It seems like exceptions thrown by observers shouldn't fail user code

Reply:
I'm 50/50 on this one, since I'm not completely the users of this function should be considered users. Closer description would be developers, which can probably handle just find the exception.